tailwind.config.js

```diff
@@ -10,7 +10,7 @@ export default {
   darkMode: 'selector',
   theme: {
     screens: {
-      sm: '428px',
```

Comment:
what is the reason to change this for 3 px?

Reply:
I have reasons...